doc/specs/#1625 - Android app support

```diff
@@ -0,0 +1,113 @@
+
+## Inspiration
+
+Android is by far the most used operating system on the planet, excluding a few certain regions, where iOS dominates. Many mobile developers make apps for Android, and only have a sometimes inferior web app for desktop, if they have one at all.
+
+## Solution Design
+
+### System requirements
+
+Windows 11 or newer  
+Windows Subsystem for Android
+ADB (if it ends up being the method of installation)
+
+### Android YAML manifest
+
+The "InstallerType" field in the YAML manifest file specifies the type of the application. A new InstallerType "Android" or "APK" will be added to show that a given application is an Android app.
+
+Each PWA would have its own manifest YAML file. A sample manifest file would look like this:
+
+```
+Id: FDroid.FDroid
+Version: 1337.0
+Name: F-Droid
+Publisher: F-Droid
+AppMoniker: fdroid
+License: Test
+InstallerType: Android
+Installers:
+- Arch: neutral
+  Url: https://f-droid.org/F-Droid.apk
+  InstallerType: Android
+  MinAndroidPlat: 31
+  ADBFlags: 
+ManifestVersion: 0.1.0
+```
+
+### Package generation
+
+WinGetCreate will download the package and extract metadata from AndroidManifest.xml, automatically filling out all data excluding the short desciption
+
+### Package installation
+
+Either ADB could be used to install the application, or the method that Microsoft Store installs Android apps could be utilized.
+### Flow of the install process
+
+1. Windows Subsystem for Android is installed
+2. ADB must be installed on the machine
+3. It must connect to the Subsystem, bypassing the developer mode restriction, which I think the uninstallation in control panel already has.
+A nice feature in the manifest would be for adding the adb install [flags](https://adbshell.com/commands/adb-install) for installation of apps that require it.
+## UI/UX Design
+
+Installing a PWA will be similar to installing other application types currently supported by winget, using the install command. The -i flag will not be implemented due to it's impracticality.
+
+An example of the interactive install would look like this:
+
+```
+>winget install fdroid
+Found F-Droid [FDroid.FDroid]
+This application is licensed to you by its owner.
+Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
+Starting Windows Subsystem for Android...
+Connecting to Windows Subsystem for Android via adb...
+Starting package install...
+<Progress bar> (if ADB supports)
+Successfully installed!
+```
+
+## Capabilities
+
+# A field to determine minumum platform SDK version of Android is required for the app to run on like 27 (Oreo), 30 (Red Velvet Cake), 31 (Snow Cone), 32 (12L), or 33 (Tiramisu)
+# A field for ADB flags to be put, like test apps.
+### Accessibility
+
+This should have no direct impact on accessibility.
+
+### Security
+
+Android has security concerns.
```

Comment:
How exactly would this featue impact security, more specifically than the blanket statement that there are security concerns?